doc/getting_started/build_sw.md

```diff
@@ -221,9 +221,9 @@ Bazel produces various artifacts depending on the category of software that is b
 
 Device software is built and run on OpenTitan hardware.
 There are three OpenTitan "devices" for simulating/emulating OpenTitan hardware:
-1. DV simulation (i.e., RTL simulation with commercial simulators),
-1. Verilator simulation (i.e., RTL simulation with the open source Verilator simulator),
-1. FPGA.
+1. "sim\_dv" DV simulation (i.e., RTL simulation with commercial simulators),
```

Comment:
Nit: I think this could use some punctuation between terms, e.g. "sim\_dv" and their definitions.